src/main/java/seedu/address/storage/JsonTipsStorage.java

```diff
@@ -16,7 +19,15 @@
 public class JsonTipsStorage implements TipsStorage {
 
     private static InputStream fileStream = JsonTipsStorage.class.getResourceAsStream("/json/tips.json");
+    private static Path filePath;
```

Comment:
I just removed this filePath, since its was mainly for logging purposes and caused a lot of build issues.
The static initialiser also caused the same build issue.